assets/js/workflow-reducer.js

```diff
@@ -58,6 +59,39 @@ const registerReducerFunc = (key, func) => {
   reducerFunc[key] = func;
 };
 
+const nonces = [];
```

Comment:
Why do you keep an array of nonces? The entire point of long random strings is that they won't collide.